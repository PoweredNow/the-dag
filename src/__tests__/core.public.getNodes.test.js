const addEdges = require('../core/public/addEdges');
const addNodes = require('../core/public/addNodes');
const getNodes = require('../core/public/getNodes');

// const { getState } = require('../core/private/_state');

describe('core.public.getNodes', () => {
  test('exports', () => {
    expect(getNodes).toMatchSnapshot();
  });
  test('Gets edges', () => {
    addNodes([{ nodeID: 1 }, { nodeID: 2 }]);
    addEdges([{ source: { nodeID: 1 }, target: { nodeID: 2 } }]);
    expect(getNodes()).toMatchSnapshot();
    // expect(state).toMatchSnapshot();
  });
});
