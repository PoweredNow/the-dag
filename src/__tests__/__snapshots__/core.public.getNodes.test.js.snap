// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core.public.getNodes Gets edges 1`] = `
Object {
  "1": Object {
    "nodeData": undefined,
    "nodeID": 1,
    "possibleTargets": Array [
      2,
    ],
  },
  "2": Object {
    "nodeData": undefined,
    "nodeID": 2,
    "possibleSources": Array [
      1,
    ],
    "possibleTargets": Array [],
  },
}
`;

exports[`core.public.getNodes exports 1`] = `[Function]`;
